### PR TITLE
docs: add Covalent Python and Typescript SDK info on docs page 

### PR DESCRIPTION
**Pull request by Dennayz**
## Description

The PR intends to describe Covalent's SDK packages (Python and TypeScript) and our backend API under the respectable documentation sections. This will go along with the recent covalent Golang SDK PR https://github.com/ethereum/ethereum-org-website/pull/12543.

### CODE REVIEW

**Comment by wackerow** [comment]
Thanks @Dennayz! Flagged for product review

**Comment by Dennayz** [comment]
> Thanks @Dennayz! Flagged for product review

Thanks! I also fixed the merge conflict! :)

**Comment by Dennayz** [comment]
@wackerow Hey! Just wondering if there are any updates on this? Thanks!

**Comment by wackerow** [comment]
Hey @Dennayz, question.. was checking out https://www.npmjs.com/package/@covalenthq/client-sdk and tried to look at the repo but it 404's.. has it moved or is it closed?

**Comment by Dennayz** [comment]
> Hey @Dennayz, question.. was checking out https://www.npmjs.com/package/@covalenthq/client-sdk and tried to look at the repo but it 404's.. has it moved or is it closed?

Hey @wackerow, thanks for the reply! I think the repo is private.

**Comment by wackerow** [comment]
Thanks @Dennayz.. curious if you know if there are plans to open source the SDK in the future? To my knowledge the other JS and Python products on two of the pages here are all open source. We could probably add this to the frameworks page, but hesitant to open the others up to closed source.
cc: @minimalsm @corwintines curious if you have an opinion here

**Comment by Dennayz** [comment]
> Thanks @Dennayz.. curious if you know if there are plans to open source the SDK in the future? To my knowledge the other JS and Python products on two of the pages here are all open source. We could probably add this to the frameworks page, but hesitant to open the others up to closed source. cc: @minimalsm @corwintines curious if you have an opinion here

Hey @wackerow, thanks for the message, I do not think these SDK's will be open sourced anytime soon.

**Comment by wackerow** [comment]
@Dennayz Thoughts on starting with a listing on the Frameworks page only for this, given the open source aspect of the JS/Python lists?

File: public/content/developers/docs/programming-languages/python/index.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ Need a more basic primer first? Check out [ethereum.org/learn](/learn/) or [ethe
 - [pymaker](https://github.com/makerdao/pymaker) - _Python API for Maker contracts_
 - [siwe](https://github.com/spruceid/siwe-py) - _Sign in with Ethereum (siwe) for Python_
 - [Web3 DeFi for Ethereum integrations](https://github.com/tradingstrategy-ai/web3-ethereum-defi) - _A Python package with ready integrations for ERC-20, Uniswap and other popular projects_
+- [Covalent Python SDK](https://pypi.org/project/covalent-api-sdk/) - _Efficient blockchain data access via Python SDK for 200+ blockchains_
```
Comment (wackerow) [inline public/content/developers/docs/programming-languages/python/index.md:61]:
```suggestion
```

File: public/content/developers/docs/apis/javascript/index.md [hunk 2]
```diff
@@ -279,6 +279,13 @@ ethers.utils.formatEther(balance)
 - [Documentation](https://viem.sh)
 - [GitHub](https://github.com/wagmi-dev/viem)
 
+**Covalent -** **_TypeScript SDK for fetching 200+ blockchain data, including token balances, nft metadata, transactions data and more._**
+
+- [covalenthq.com](https://www.covalenthq.com/)
+- [Documentation](https://www.npmjs.com/package/@covalenthq/client-sdk)
+- [GitHub](https://github.com/covalenthq)
+- [Discord](https://www.covalenthq.com/discord/)
+
```
Comment (wackerow) [inline public/content/developers/docs/apis/javascript/index.md:288]:
```suggestion
```

**Review by wackerow — CHANGES_REQUESTED** [review]

**Review by wackerow — APPROVED** [review]
Removed from JS and Python pages at least for now, but left in for the frameworks page. Thanks @Dennayz! Bringing in

**Comment by wackerow** [comment]
@all-contributors please add @Dennayz for tool

**Pull request merged**